plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/expression/SparkExpressions.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.deltalake.expression;
+
+import com.google.common.annotations.VisibleForTesting;
+import io.trino.spi.TrinoException;
+import org.antlr.v4.runtime.BaseErrorListener;
+import org.antlr.v4.runtime.CharStreams;
+import org.antlr.v4.runtime.CommonTokenStream;
+import org.antlr.v4.runtime.ParserRuleContext;
+import org.antlr.v4.runtime.RecognitionException;
+import org.antlr.v4.runtime.Recognizer;
+import org.antlr.v4.runtime.atn.PredictionMode;
+import org.antlr.v4.runtime.misc.ParseCancellationException;
+
+import java.util.function.Function;
+
+import static io.trino.spi.StandardErrorCode.NOT_SUPPORTED;
+
+public final class SparkExpressions
```

Comment:
Can call this class `SparkExpressionParser`?

the antlr generated class would want to be called eg `SparkExpressionBaseParser`
(just rename grammar file and  update `grammar SparkExpression;` line)